src/kubectl-object-model.ts

```diff
@@ -9,10 +9,12 @@ export class Kubectl {
     private kubectlPath: string;
     private namespace: string;
     private ignoreSSLErrors: boolean;
+    private forceDeployment: boolean;
 
-    constructor(kubectlPath: string, namespace?: string, ignoreSSLErrors?: boolean) {
+    constructor(kubectlPath: string, namespace?: string, ignoreSSLErrors?: boolean, forceDeployment?: boolean) {

```

Comment:
Also, given that this is only to be used for `apply` doesn't it make more sense to be a parameter to that function instead?